modules/azure/core/routes.tf

```diff
@@ -23,6 +24,20 @@ resource "azurerm_route" "this" {
   next_hop_in_ip_address = each.value.route.next_hop_in_ip_address
 }
 
+resource "azurerm_route" "not_virtual_appliance" {
```

Comment:
Please add a note about how we should add move commands in the future when we upgrade to Terraform 1.1.0.
https://github.com/hashicorp/terraform/releases